Makefile

```diff
@@ -552,6 +555,50 @@ verify-examples:
 	$(diff) -r '$(tmp_dir)'/ ./examples
 .PHONY: verify-examples
 
+# $1 - dashboard dir
+# $2 - output configmap location
+define embed-dashboard
+	if [[ ! -d '$(2)' ]]; then mkdir '$(2)'; fi
+	cp -r '$(1)'/*.json '$(2)'
+
+endef
+
+# $1 - monitoring assets dir
+define embed-dashboards
+	$(foreach d,$(wildcard $(MONITORING_DASHBOARDS_DIR)/ver_*),$(call embed-dashboard,$(d),$(1)/$(subst ver_,scylladb-,$(notdir $(d)))))
+	$(foreach d,$(wildcard $(MONITORING_DASHBOARDS_DIR)/manager_*),$(call embed-dashboard,$(d),$(1)/$(subst manager_,manager-,$(notdir $(d)))))
+
+endef
+
+# $1 - rules assets dir
+define embed-rules
+	cp '$(MONITORING_RULES_DIR)'/*.yml '$(1)'/
+
+endef
+
+# $1 - rules assets dir
+define embed-monitoring
+	$(RM) -r '$(1)/grafana/v1alpha1/dashboards/platform/'*/
+	$(RM) -r '$(1)/prometheus/v1/rules/'*/
+	$(call embed-dashboards,$(1)/grafana/v1alpha1/dashboards/platform)
+	$(call embed-rules,$(1)/prometheus/v1/rules)
+
+endef
+
+update-monitoring: submodules
+	$(call embed-monitoring,./assets/monitoring)
+.PHONY: update-monitoring
+
+verify-monitoring: tmp_dir :=$(shell mktemp -d)
+verify-monitoring: submodules
```

Comment:
there is no `submodules` target?

Reply:
looks like that got lost, thx
(it's just the init)

Reply:
added